docs/backend/control-panels.md

```diff
@@ -0,0 +1,42 @@
+---
+html_meta:
+  "description": "How to add a Control Panel"
+  "property=og:description": "How to add a Control Panel"
+  "property=og:title": "Add Control Panel"
+  "keywords": "Plone, Add, Control Panel"
+---
+
+(backend-controlpanels-label)=
+
+# Control panels
+
+## Adding a control panel
+To add a control panel to your add-on, you can use [`plonecli`](https://pypi.org/project/plonecli/) as follows:
```

Comment:
A blank line should exist before and after every heading level for author readability.